librecad/src/lib/engine/rs_ellipse.cpp

```diff
@@ -1324,7 +1324,7 @@ const RS_EllipseData& RS_Ellipse::getData() const
  * x *= kx
  * y *= ky
  * find the maximum and minimum of x^2 + y^2,
- */
```

Comment:
To do carry out testing, please keep both methods(renaming them?)

Try something like, (just ideas, not correct at coding)

cout << ellipse.oldscale().getangle()-ellipse.scale().getangle()


cout << ellipse.oldscale().geratio()-ellipse.scale().geratio()